ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -424,15 +424,16 @@ tests:
     - chain: cucushift-installer-check-cluster-health
   timeout: 4h0m0s
 - always_run: false
-  as: e2e-aws-ovn-user-provisioned-dns
+  as: e2e-aws-custom-dns-techpreview
```

Comment:
@yunjiang29 per last discussion with @jianlinliu , we currently prefer to use featureGate to enable AWS Custom DNS feature to filter out such tests in jobs easily. What do you think?
```
      FEATURE_GATES: '["AWSClusterHostedDNS=true"]'
      FEATURE_SET: CustomNoUpgrade
```